google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClientFactory.java

```diff
@@ -78,7 +79,9 @@ public final class BigtableDataClientFactory implements AutoCloseable {
    */
   public static BigtableDataClientFactory create(BigtableDataSettings defaultSettings)
       throws IOException {
-    ClientContext sharedClientContext = ClientContext.create(defaultSettings.getStubSettings());
+    EnhancedBigtableStubSettings settings =
+        EnhancedBigtableStub.finalizeTransportSettings(defaultSettings.getStubSettings());
```

Comment:
I wonder if it would be cleaner to instead have a static method on EnhancedBigtableStub to create the ClientContext instead of mucking with settings?